test/plugin/scenarios/spring-async-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+  applications:
+    - {spring-async-scenario: 2}
+  instances:
+    - {spring-async-scenario: 1}
+  operationNames:
+    - spring-async-scenario: [/spring-async-scenario/case/healthCheck, /spring-async-scenario/case/spring-async,
+                              /]
+  heartbeat: []
+segmentItems:
+  - applicationCode: spring-async-scenario
+    segmentSize: 2
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: /spring-async-scenario/case/healthCheck
+            operationId: 0
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 1
+            componentName: ''
+            isError: false
+            spanType: Entry
+            peer: ''
+            peerId: 0
+            tags:
+              - {key: url, value: 'http://localhost:8080/spring-async-scenario/case/healthCheck'}
+              - {key: http.method, value: HEAD}
+      - segmentId: not null
+        spans:
+          - operationName: /
+            operationId: 0
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 2
+            componentName: ''
+            isError: false
+            spanType: Exit
+            peer: skywalking.apache.org:80
+            peerId: 0
+            tags:
+              - {key: url, value: 'http://skywalking.apache.org/?k=v'}
+              - {key: http.method, value: GET}
+          - operationName: /
```

Comment:
Maybe the old case is not right either. This span is inside the entry span(`/spring-async-scenario/case/spring-async`), so it is showing async. Please recheck.

Reply:
done